src/components/Alert/styles/AlertBorder.js

```diff
@@ -3,15 +3,18 @@ import get from 'extensions/themeGet';
 
 const AlertBorder = styled.div`
   padding: ${get('spacing.small')} ${get('spacing.medium')};
-  font-weight: 500;
+  font-weight: 200;
   border-width: ${get('thicknesses.normal')};
   border-style: solid;
```

Comment:
I noticed when working locally with this branch that the `border-color` is not defined, which means border will take its color from the `color` css attribute by default. That was probably an oversight. According to the guidelines, it should have an explicit color based on alert type, `#00bef0` (`colors.primary.default`) for info, `#00fbb9` (I actually think we should replace `colors.positive.border` with that value) for success, `#ffb39e` (again, replace `colors.negative.border` with this value) for error.

For context, the values in the theme were taken by me from our company's official brand guides verbatim. I think it's proper for us to replace them with new values when we get them from UX.